openedx/features/edly/admin.py

```diff
@@ -49,9 +49,25 @@ class EdlyOrganizationAdmin(admin.ModelAdmin):
     """
     Admin interface for the EdlyOrganization object.
     """
+    search_fields = ['name', 'slug']
     list_display = ['name', 'slug', 'created', 'modified']
     inlines = [EdlySubOrganizationInlineAdmin]
 
 
+class EdlyUserProfileAdmin(admin.ModelAdmin):
+    """
+    Admin interface for the EdlyUserProfile object.
```

Comment:
Use double quotes to mention name of methods/classes
```python
Admin interface for the "EdlyUserProfile" object.
```

Reply:
Done